packs/typeform/sensors/registration_sensor.py

```diff
@@ -92,7 +94,19 @@ def _get_api_registrations(self, params):
         headers = {}
         headers['Content-Type'] = 'application/x-www-form-urlencoded'
 
-        response = requests.get(url=self.url, headers=headers, params=data)
+        for x in range(self.sensor_config['retries']):
+            try:
+                response = requests.request(
+                    method='GET',
+                    url=self.url,
+                    headers=headers,
+                    timeout=self.sensor_config['timeout'],
+                    params=data)
+            except Exception, e:
+                self.logger.info(e)
```

Comment:
After a failure it is typical to wait for a bit before the retry. It is unlikely that a network/intermittent error would not resolved almost immediately.